README.md

```diff
@@ -2,34 +2,41 @@
 
 [Raygun](https://raygun.com/) provider for iOS, tvOS & macOS supporting Crash Reporting and Real User Monitoring.
 
-Supports iOS 10+, tvOS 10+ & macOS 10.10+
+Supports:
+- iOS 10+
+- tvOS 10+
+- macOS 10.10+
 
 ## Installation
 
-### With package installer
-
-The latest version can be downloaded [here](https://downloads.raygun.com/Raygun4Apple/latest/raygun4apple.pkg). Once downloaded, run the installer and follow the on-screen instructions. The frameworks will be installed onto your local machine under  *Library > Frameworks > raygun4apple*.
-
-Once installed, go to your app's target **General** settings and add the raygun4apple framework to the **Frameworks, Libraries, and Embedded Content** section. Ensure that the framework is set to **Embed & Sign**.
-
 ### With GitHub releases
 
 The latest release can be found [here](https://github.com/MindscapeHQ/raygun4apple/releases). The frameworks are attached to each release as a zipped file. This can be downloaded, unzipped and included in you project directory.
 
 Once included, go to your app's target **General** settings and add the raygun4apple framework to the **Frameworks, Libraries, and Embedded Content** section. Ensure that the framework is set to **Embed & Sign**.
 
+### With CocoaPods
```

Comment:
Suggestion: Would this be considered the preferred approach? If so, it could be listed above the GitHub steps.

Reply:
That's a good idea as it will be the preferred way.